src/components/Header/Pages.js

```diff
@@ -35,6 +38,7 @@ const playerPages = [{
   name: 'Counts',
 }, {
   name: 'Histograms',
+  extra: histogramNames[0],
```

Comment:
Instead of rerouting, what if you just treat empty string (/histograms) as the default internally without changing the URL?  (this is the current behavior)